lib/membrane_opus/parser/delimitation.ex

```diff
@@ -0,0 +1,77 @@
+defmodule Membrane.Opus.Parser.Delimitation do
+  @moduledoc false
+  # Helper module for delimiting or undelimiting packets
+
+  @type delimitation_t :: :delimit | :undelimit | :keep
+  @type handler_t :: __MODULE__.Undelimiter | __MODULE__.Delimiter | __MODULE__.Keeper
+
+  @spec get_handler(
+          delimitation :: delimitation_t(),
+          self_delimiting? :: boolean
+        ) ::
+          {handler :: handler_t, self_delimiting? :: boolean}
+  def get_handler(delimitation, self_delimiting?) do
+    cond do
+      self_delimiting? && delimitation == :undelimit ->
+        {__MODULE__.Undelimiter, false}
+
+      !self_delimiting? && delimitation == :delimit ->
+        {__MODULE__.Delimiter, true}
+
+      true ->
+        {__MODULE__.Keeper, self_delimiting?}
+    end
+  end
+
+  defmodule Handler do
```

Comment:
`moduledoc false` should be also placed in all nested modules. Please check `mix docs`